tests/resources/docker/docker-compose.yml

```diff
@@ -266,6 +266,34 @@ services:
       - _APP_REDIS_PORT
       - _APP_SMTP_HOST
       - _APP_SMTP_PORT
+  
+  appwrite-worker-builds:
```

Comment:
We don't really need this here. This file is only used to test our docker compose parser class.